cmd/swapd/main.go

```diff
@@ -31,11 +33,6 @@ const (
 	defaultXMRTakerLibp2pPort = 9933
 	defaultXMRMakerLibp2pPort = 9934
 
-	// default libp2p key files
-	defaultLibp2pKey         = "node.key"
-	defaultXMRTakerLibp2pKey = "xmrtaker.key"
-	defaultXMRMakerLibp2pKey = "xmrmaker.key"
```

Comment:
We now place the libp2p key in `{DATA_DIR}/net.key`.  The XMR taker and maker use separate data dirs eliminating the need for these.